akka-http-core/src/main/scala/akka/http/scaladsl/model/headers/headers.scala

```diff
@@ -57,9 +60,6 @@ sealed trait ModeledHeader extends HttpHeader with Serializable {
  * as they allow the custom header to be matched from [[RawHeader]] and vice-versa.
  */
 abstract class CustomHeader extends jm.headers.CustomHeader {
-  /** Override to return true if this header shouldn't be rendered */
-  def suppressRendering: Boolean = false
-
```

Comment:
So this now makes people to explicitly say if it should be rendered in request/response right?
Do we need to make everyone think about this?